framework-cloud/framework-git/src/main/java/org/kie/cloud/git/constants/GitConstants.java

```diff
@@ -52,6 +60,27 @@ public static String getGitHubPassword() {
         return System.getProperty(GITHUB_PASSWORD);
     }
 
+    public static String getGogsUrl() {
+        return System.getProperty(GOGS_URL);
+    }
+
+    public static String getGogsUser() {
+        return System.getProperty(GOGS_USER);
+    }
+
+    public static String getGogsPassword() {
+        return System.getProperty(GOGS_PASSWORD);
+    }
+
+    public static String readMandatoryParameter(String value, String parameterName) {
```

Comment:
Wouldn't it be better to make this implementation detail of each of the methods? I don't like that the API doesn't enforce the "mandatoriness". As it is you can misuse it by calling `GitConstants.getGogsPassword`and it would fail if not seat. How about this instead?
```
public static String getGogsPassword() {
       return readMandatoryParameter(GOGS_PASSWORD);
}

private static String readMandatoryParameter(String parameterName) {
        Strinv value = System.getProperty(parameterName)
        if (value == null) {
            logger.error("Parameter {} must be specified", parameterName);
            throw new RuntimeException("Parameter " + parameterName + " must be specified");
        }
        return value;
 }
```

Reply:
@jhrcek Yes, you are right. But it is problem of all parameters in Kie cloud tests. I think we can polish it in another PR.